Data/translations/Covid Localization - Home Page.csv

```diff
@@ -153,12 +153,25 @@ definitions_link,## {#defintions},## {#defintions}
 DT_label_double,doubling time,temps de doublement
 DT_label_half,halving time,temps de division
 DT_label_no_change,no growth (flat),pas de croissance
+end_transition_out,"**Note:** June 1, 2023
+
+613covid as a public-facing tool has been a point of tremendous pride for our lab. We are very happy that we were able to generate something quickly, with the support and collaboration of our colleagues across disciplines in Ottawa and Ontario. The website has continued to report data on the SARS-CoV-2 signal in wastewater for several years, and has expanded to even include reporting of influenza and RSV viruses. We are so honoured to have so many people in Ottawa use, consult, and support this application of ours. 
```

Comment:
'tremendous pride for the Ottawa Covid-19 collaboration between the uOttawa Delatolla lab, the Ottawa Hospital, CHEO and Ottawa Public Health.... We are so honoured to have so many people in Ottawa use, consult, and support this application of ours. There have been over 2 million users.